app/models/todo.rb

```diff
@@ -1,2 +1,8 @@
 class Todo < ApplicationRecord
+  validates :title, presence:true
```

Comment:
Un espace entre le symbole `presence:` et le boolean `true` est préférable.

```suggestion
  validates :title, presence: true
```